src/Application/Gallery/Gallery.tsx

```diff
@@ -63,16 +63,12 @@ class Gallery extends React.Component<Props, State> {
     }
   }
 
-  componentDidUpdate(prevProps: Props, nextState: State) {
-    if (!nextState.visible) {
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    if (!prevState.visible && this.state.visible) {
```

Comment:
Is this to check if the the state has moved from not visible to visible  for the focused container on update?

Reply:
yes, once the Slider is mounted, it should be focused to make sure the arrow keys can control it.

Reply:
Btw, I added more conditions to ensure the focus event won't cause a reference error.